sdk/examples/xo_python/sawtooth_xo/xo_cli.py

```diff
@@ -170,10 +176,16 @@ def do_init(args, config):
     if args.url is not None:
         url = args.url
 
+    user = config.get('DEFAULT', 'user')
+    if args.user is not None:
+        user = args.user
```

Comment:
At this point 'user' is the complete auth header information "USERNAME:[PASSWORD]" not just the user.  Suggest immediately breaking this into auth_user and auth_password here for clarity.

If auth_user is specified and no auth_password is specified, the user should be prompted every time xo commands are run for the password, with getpass.getpass().